Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/Microsoft.Toolkit.Win32.UI.Controls.WPF.csproj

```diff
@@ -11,20 +11,6 @@
     <Description>This library provides XAML user controls for WPF. It is part of the Windows Community Toolkit.</Description>
     <PackageTags>WebView Win32 WPF</PackageTags>
 
-    <!-- Need Metadata contract 6.0 or later -->
-    <!-- First introduced in 17110: Insider Preview (Fast) -->
-    <!-- Most people will have RTM 17134: April 2018 Update -->
-    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
-    <!--
```

Comment:
Could you also move this comment to the directory.build.props file?

Reply:
Will do.   @ryalanms, do you have time to address the formatting issues with the Interop.WPF & Interop.WinForms code, or would it be helpful for me to take a crack at it?

Reply:
I’ve already done some formatting fixes in my fork

Reply:
See https://github.com/rjmurillo/UWPCommunityToolkit/tree/rimuri/xaml-islands_more-fixes; 6e4504014bf461aa280b10d8051db20764012722 40e92379c0c1b5c251318a9fe6c45e29dc5f562c 1d01a7d62769d2626e5bcc12acbb8730ba230b2c e529f2e2a78d9af69d4d3d276968dd4684c8d9fd

Reply:
Awesome, thanks!

Reply:
Just made it a PR (#2322) for easier review